src/services/location.js

```
@@ -27,10 +33,20 @@ const getCitiesByCountry = async (countryName) => {
     }
   }
 
+  let filteredCities = country.cities
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)